### DISCUSSION TITLE
Run Selenide Test in Docker container (Page Oobject Model Framework)

### DISCUSSION THREAD

**Discussion by georgerobertkplivi** [selenide/selenide#2004] [General]
I have been trying to run selenide test in docker but I have run out of ideas. I am new to this beautiful Library. Thank you
```java
import org.junit.Before;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeOptions;
import org.openqa.selenium.remote.RemoteWebDriver;

import java.net.MalformedURLException;
import java.net.URL;

import static com.codeborne.selenide.Selenide.open;

public class BaseTest {

    final String baseURL = "https://www.amazon.in/";

    @Before
    public void setUp() throws MalformedURLException {

        ChromeOptions options = new ChromeOptions();
        URL remoteURL = new URL("http://localhost:4444/wd/hub");
        WebDriver driver = new RemoteWebDriver(remoteURL, options);
        driver.get(baseURL);

//        open(baseURL);
    }
}
```

**Comment by asolntsev** [comment]
@georgerobertkplivi Hi. Welcome to the club! :) 

First of all, please describe your problem. What are you trying to do; what is the expected and actual result. Are you getting some error? Send the full stacktrace.

**Reply by georgerobertkplivi** [reply]
Thank you for the response.
I want my test to run inside of docker with Selenide as the selenium framework but it isn't working for me with the above code. Thank you

**Reply by georgerobertkplivi** [reply]
All my page Object class I coded with selenide

**Reply by asolntsev** [reply]
Once again: what means "isn't working"? Are you getting some error? Send the full stacktrace.
